msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -27,7 +27,7 @@ Tabs allow team members to access services and content in a specific space withi
 > If you have integrated a tab with your meeting, then your app must follow the Teams [single sign-on (SSO) authentication flow for tabs](../tabs/how-to/authentication/auth-aad-sso.md).
 
 > [!NOTE]
-> Apps are supported in private scheduled meetings only.
+> Apps are only supported in private scheduled meetings and standard channel meetings.
```

Comment:
please remove `scheduled`. 
In private meeting, we support both scheduled & recurring meetings.

Reply:
@YunnyChung , aren't the recurring meeting scheduled meetings?!

@surbhigupta12, can we say "scheduled channel meetings" instead of "standard channel meetings" in consistency with Teams Client UI